subprojects/test-runner/device-provider/impl/src/main/kotlin/com/avito/android/runner/devices/internal/kubernetes/ReservationDeploymentFactoryImpl.kt

```diff
@@ -103,41 +100,43 @@ internal class ReservationDeploymentFactoryImpl(
                         }
                     }
 
-                    if (emulator.gpu) {
-                        volumeMounts = listOf(
-                            newVolumeMount {
-                                name = "x-11"
-                                mountPath = "/tmp/.X11-unix"
-                                readOnly = true
-                            }
-                        )
-
-                        env = listOf(
-                            newEnvVar {
-                                name = "GPU_ENABLED"
-                                value = "true"
-                            },
-                            newEnvVar {
-                                name = "SNAPSHOT_DISABLED"
-                                value = "true"
-                            }
-                        )
-                    }
+                    env = listOf(
+                        // used to start from prepared snapshot, see `prepare_snapshot.sh`
+                        newEnvVar {
+                            name = "SNAPSHOT_ENABLED"
+                            value = "true"
+                        },
+                        // run headless
+                        newEnvVar {
+                            name = "WINDOW"
+                            value = "false"
+                        }
+                    )
+
+                    /* could be mounted for remote debugging
```

Comment:
The code is no place for dead code
Let's move it to documentation

Reply:
Or hide by disabled feature toggle

Reply:
removed completely for now, i don't know if it's even supported now